cmake/templates/OpenVINOConfig.cmake.in

```diff
@@ -283,6 +283,18 @@ if(NOT TARGET openvino)
     set(_ov_as_external_package ON)
     include("${CMAKE_CURRENT_LIST_DIR}/OpenVINOTargets.cmake")
 
+    foreach(target openvino::runtime openvino::runtime::c
+               openvino::frontend::onnx openvino::frontend::paddle openvino::frontend::tensorflow
+               openvino::frontend::pytorch openvino::frontend::tensorflow_lite)
+        if(TARGET ${target})
+            get_target_property(ORIGINAL_NAME ${target} ALIASED_TARGET)
+            if (ORIGINAL_NAME MATCHES "ORIGINAL_NAME-NOTFOUND")
+                set (ORIGINAL_NAME ${target})
+            endif()
+            set_property(TARGET ${ORIGINAL_NAME} PROPERTY "MAP_IMPORTED_CONFIG_RELWITHDEBINFO" "RELEASE")
```

Comment:
what if RelWithDebInfo is installed? 
We need to check whether RELWITHDEBINFO is among of [IMPORTED_CONFIGURATIONS](https://cmake.org/cmake/help/latest/prop_tgt/IMPORTED_CONFIGURATIONS.html)

Reply:
done